platform/x11/power_x11.cpp

```diff
@@ -115,7 +115,7 @@ bool PowerX11::make_proc_acpi_key_val(char **_ptr, char **_key, char **_val) {
 
 	*(ptr++) = '\0'; /* terminate the key. */
 
-	while ((*ptr == ' ') && (*ptr != '\0')) {
+	while ((*ptr != ' ') && (*ptr != '\0')) {
```

Comment:
Shouldn't the loop run until it hits a non-space character? 🤔 
In the following lines there is an explicit check for `'\0'` so the additional check in the loop can be removed, I think:
https://github.com/godotengine/godot/blob/5ce3999a77dd94f099c49ab841364d6cdf7c630b/platform/x11/power_x11.cpp#L122-L124
So the fix might look like
```suggestion
	while (*ptr == ' ') {
```

Reply:
Right, I wrongly understand comment.